tests/VolleyManagement.UnitTests/Services/GameService/GameServiceTests.cs

```diff
@@ -1648,6 +1648,80 @@ public void Edit_GameValid_GameEdited()
             VerifyEditGame(game, Times.Once());
         }
 
+        /// <summary>
+        /// Test for Edit method. Change time for game in second round with unknown teams. 
+        /// No exceprtion returns and game is edited successfully.
+        /// </summary>
+        [TestMethod]
+        public void Edit_ChangeTimeForPlayOffGameWithUnknownTeams_GameSaved()
+        {
+            //Arrange
+            bool isExceprionThrown = false;
+            var testTournament = CreatePlayoffTournament();
+            var games = new GameTestFixture()
+                .TestEmptyGamePlayoffSchedule()
+                .Build();
+
+            var testGameForEdit = games[5];
```

Comment:
Why 5?

Reply:
Because teams in the first round are already known and it worked good in all cases.
So we choose game in second round with unknown teams.